src/main/resources/jenkins/branch/BranchNameContributor/buildEnv.properties

```diff
@@ -21,6 +21,7 @@
 # THE SOFTWARE.
 
 blurb.BRANCH_NAME=For a multibranch project, this will be set to the name of the branch being built, for example in case you wish to deploy to production from <code>master</code> but not from feature branches; if corresponding to some kind of change request, the name is generally arbitrary (refer to <code>CHANGE_ID</code> and <code>CHANGE_TARGET</code>).
+blurb.BRANCH_IS_PRIMARY=For a multibranch project, this will be set to <code>true</code> if the branch being built is the primary branch of the SCM source; otherwise, it is set to <code>false</code>.
```

Comment:
```suggestion
blurb.BRANCH_IS_PRIMARY=For a multibranch project, if the SCM source reports that the branch being built is a primary branch, this will be set to <code>true</code>, otherwise this will be set to <code>false</code>. Some SCM sources may report more than one branch as a primary branch while others may not supply this information. 
```